gluoncv/utils/metrics/segmentation.py

```diff
@@ -74,7 +74,7 @@ def batch_pix_accuracy(output, target):
     """PixAcc"""
     # inputs are NDarray, output 4D, target 3D
     # the category -1 is ignored class, typically for background / boundary
-    predict = np.argmax(output.asnumpy().astype('int64'), 1) + 1
```

Comment:
@zhanghang1989 Thanks a lot to you for your great contribution to this community. Could you please explain why the data type of **output** in the code "_predict = np.argmax(output.asnumpy().astype('int64'), 1) + 1_" changes to int64? It confuses me for a long time. Thanks in advance.